Documentation/ceph-csi-snapshot.md

```diff
@@ -0,0 +1,101 @@
+---
+title: ceph-csi Snapshots
```

Comment:
Is there another type of snapshots besides ceph-csi? This will be more consistent with the table of contents.
```suggestion
title: Snapshots
```

Reply:
Don't we have/get any other storage provider which supports csi in rook? If no I can change it

Reply:
This topic is in the Ceph section of the TOC, so there is no need for ceph-specific prefixes.

Reply:
Okay will update it then

Reply:
@travisn done